omod-1.9/src/main/java/org/openmrs/module/webservices/rest/web/v1_0/resource/openmrs1_9/ConceptStopwordResource1_9.java

```diff
@@ -0,0 +1,132 @@
+        if (rep instanceof RefRepresentation || rep instanceof DefaultRepresentation) {
+            DelegatingResourceDescription description = new DelegatingResourceDescription();
+            description.addProperty("uuid");
+            description.addProperty("value");
+            description.addProperty("locale");
+            description.addSelfLink();
+            description.addLink("full", ".?v=" + RestConstants.REPRESENTATION_FULL);
+            return description;
+        } else if (rep instanceof FullRepresentation) {
+            DelegatingResourceDescription description = new DelegatingResourceDescription();
+            description.addProperty("uuid");
+            description.addProperty("value");
+            description.addProperty("locale");
+            description.addSelfLink();
+            return description;
+        }
+        return null;
+    }
+
+    @PropertyGetter("value")
+    public String getDisplayString(ConceptStopWord delegate) {
+        return StringUtils.isEmpty(delegate.getValue()) ? "" : delegate.getValue();
+    }
+
+    /**
+     * @see org.openmrs.module.webservices.rest.web.resource.impl.BaseDelegatingResource#getCreatableProperties()
+     */
+     @Override
+     public DelegatingResourceDescription getCreatableProperties() {
+         DelegatingResourceDescription description = new DelegatingResourceDescription();
+         description.addRequiredProperty("value");
+         description.addProperty("locale");
+         return description;
+     }
+
+    /**
+     * @see DelegatingCrudResource#getByUniqueId(java.lang.String)
+     */
+    @Override
+    public ConceptStopWord getByUniqueId(String uniqueId) {
+        List<ConceptStopWord> datatypes = Context.getConceptService().getAllConceptStopWords();
+        for (ConceptStopWord datatype : datatypes) {
+            if (datatype.getUuid().equals(uniqueId)) {
+                return datatype;
+            }
+        }
+        return null;
+    }
+
+    /**
+     * @see DelegatingCrudResource#newDelegate()
+     */
+    @Override
+    public ConceptStopWord newDelegate() {
+        return new ConceptStopWord();
+    }
+
+    /**
+     * @see DelegatingCrudResource#save(java.lang.Object)
+     */
+    @Override
+    public ConceptStopWord save(ConceptStopWord delegate) {
+        return Context.getConceptService().saveConceptStopWord(delegate);
+    }
+
+    /**
+     * @see org.openmrs.module.webservices.rest.web.resource.impl.DelegatingCrudResource#purge(java.lang.Object,
+     *      org.openmrs.module.webservices.rest.web.RequestContext)
+     */
+    @Override
+    public void purge(ConceptStopWord delegate, RequestContext context) throws ResponseException {
+        throw new ResourceDoesNotSupportOperationException();
+    }
+
+    @Override
+    protected void delete(ConceptStopWord delegate, String reason, RequestContext context) throws ResponseException {
+        Context.getConceptService().deleteConceptStopWord(delegate.getId());
```

Comment:
Deleting doesn't work and are throwing responses with ResourceDoesNotSupportOperationException.
Thats why I "switched" delete methods between purge and delete.